src/pages/settings/Profile/Contacts/ContactMethodsPage.js

```diff
@@ -59,7 +59,9 @@ const defaultProps = {
 };
 
 const ContactMethodsPage = (props) => {
-    const loginMenuItems = _.map(props.loginList, (login, loginName) => {
+    const sortedLoginNames = _.sortBy(_.map(props.loginList, (login, loginName) => loginName), loginName => (props.loginList[loginName].partnerUserID === props.session.email ? 0 : 1));
```

Comment:
@akinwale I think we should make this line easier to read. It's currently pretty understandable if you take the time to look at each piece step by step, but it would be better if we could expand this out into a few lines and add at least 1 comment explaining what the purpose of this is

Reply:
Done.